misc/pause-container/Makefile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+.PHONY: all container clean
+
+BIN = pause
+SRCS = pause.c
+CFLAGS = -Os -Wall -Werror -static
+GCC = gcc
+
+TAG = 1.0
+IMAGE = "amazon-ecs-pause"
+
+all: container
+
+bin:
+	$(GCC) $(CFLAGS) $(SRCS) -o $(BIN)
```

Comment:
I'm talking about the build process.  I don't want to have a different version of `gcc` being used on my laptop as is being used on another contributor's development environment.  By default, we should use `gcc` inside a container so that we're using an easily-reproducible build environment.